wherehows-web/app/constants/dataset-aclaccess.ts

```diff
@@ -0,0 +1,132 @@
+import { capitalize } from '@ember/string';
+import { IpageConcent, IpageState, IAclInfo, Iprincipal } from 'wherehows-web/typings/api/datasets/aclaccess';
```

Comment:
Please capitalize the interfaces, also typo: `IPageContent`, not `IpageConcent`, same for others